gui/src/2D/xmlLoader.ts

```diff
@@ -312,24 +315,46 @@ export class XmlLoader {
      * @param rootNode defines the node / control to use as a parent for the loaded layout controls.
      * @param callback defines the callback called on layout load.
      */
-    public loadLayout(xmlFile: any, rootNode: any, callback: any): void {
+    public loadLayout(xmlFile: any, rootNode: any, onSuccess: any = null, onError: any = null): void {

```

Comment:
could you replace the type onSuccess: any by onSuccess: Nullable<() => void> and onError: any by onError : Nullable<(error: srting) => void>

Reply:
okay, done :) .